package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "AdminLTE",
+  "version": "1.0.0",
+  "private": true,
+  "description": "",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/pi-hole/AdminLTE.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```

Comment:
Not sure if this is right. Not that it matters a lot since we don't distribute the package via npm, but still